package.json

```diff
@@ -50,11 +50,11 @@
     "ember-addon"
   ],
   "dependencies": {
+    "broccoli-funnel": "1.0.2",
+    "broccoli-merge-trees": "1.1.1",
     "element-resize-detector": "1.1.4",
     "ember-cli-babel": "^5.1.6",
```

Comment:
Shouldn't this be upgraded?

Reply:
oh yeah. it doesnt actually look like this PR changes anything. I'll check it out